models/novelGenres.js

```diff
@@ -2,7 +2,7 @@ const novelGenres = (connection, Sequelize, genres, novels) => {
   return connection.define('novelGenres', {
     genreId: { type: Sequelize.INTEGER, primarykey: true, references: { model: genres, key: 'id' } },
     novelId: { type: Sequelize.INTEGER, primarykey: true, references: { model: novels, key: 'id' } },
-  }, { defaultScope: { attributes: { exclude: ['deletedAt'] } } }, { paranoid: true })
+  }, { paranoid: true })
```

Comment:
you have some named novelGenres and some named novelsGenres so it is not making your API call appropriately